### PR TITLE
Bugfix release 19.0.1

### PR DESCRIPTION
**Pull request by OleksandrVidinieiev**
Purpose: include [CIRC-768](https://issues.folio.org/browse/CIRC-768) into Q2-2020/Goldenrod release.

Resolves [CIRC-776](https://issues.folio.org/browse/CIRC-776).

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 19.0.1 2020-06-16
+* Service points with many locations do not allow pick slips to be printed (CIRC-768)
```
Comment (marcjohnson-kint) [inline NEWS.md:2]: I usually leave a gap between the heading and the description of the contents.

File: src/main/java/org/folio/circulation/resources/PickSlipsResource.java [hunk 2]
```diff
@@ -98,7 +101,7 @@ private void getMany(RoutingContext routingContext) {
     UUID servicePointId, Clients clients) {
 
     return findWithCqlQuery(clients.locationsStorage(), LOCATIONS_KEY, Location::from)
-      .findByQuery(exactMatch(PRIMARY_SERVICE_POINT_KEY, servicePointId.toString()));
+      .findByQuery(exactMatch(PRIMARY_SERVICE_POINT_KEY, servicePointId.toString()), LOCATIONS_LIMIT);
```
Comment (marcjohnson-kint) [inline src/main/java/org/folio/circulation/resources/PickSlipsResource.java:104]: Is this really a locations limit now that it's also used for service points as well?

File: NEWS.md (hunk 1 above)
Comment (OleksandrVidinieiev) [reply NEWS.md:2]: That's right. Sorry I missed that. Should I recreate this PR with correct formatting?

Comment (marcjohnson-kint) [reply NEWS.md:2]: Good question. 

No, I don't think it's worth re-doing the release. 

Maybe just push a new commit to change the formatting for the future. And change the formatting in the GitHub release notes.

Comment (OleksandrVidinieiev) [reply NEWS.md]: Pushed a fix. What do you mean by changing the formatting in the release notes? Theres is just [one line in the description](https://github.com/folio-org/mod-circulation/releases). I also created an identical PR for Flameflower: https://github.com/folio-org/mod-circulation/pull/574

Comment (marcjohnson-kint) [reply NEWS.md]: That's fine, there is no need to change that.

Comment (marcjohnson-kint) [reply NEWS.md]: Thank you

**Pull request merged**